README.md

```diff
@@ -114,3 +114,14 @@ NAME       STATUS    AGE
 10.0.0.4   Ready     20s
 
 ```
+
+## Customization
+
+User have possibility to execute custom scripts on master before and after provision process.
+
+Scripts must be stored in directory: /usr/local/scripts/ with following names:
+
+- pre-tasks.sh - executed before provision
+- post-tasks.sh - executed after provision
+
+User must add those scripts to the image with correct rights: rw-r-r
```

Comment:
I think this is best. Because I would like to keep the requirements for the prebuild image as low as possible. In best case the OS requirement is just docker (this should be our goal)

 So, I think it is better to check for directories